modules/specialized-guides/pages/large-deployments/monitoring.adoc

```diff
@@ -17,4 +17,6 @@ endif::[]
 You need to install Prometheus and Grafana on a machine separate from the {productname} Server.
 We recommend you use a managed client as your monitoring server.
 
+It's recommended to monitor {salt} event bus in the large scale deployments with xref:salt/salt-monitoring.adoc#saline[Saline] to identify possible bottlenecks.
```

Comment:
I think we need long xrefs (with main book part such as :specialized-guides:).  I do not change it everywhere in the following description, because first we must test it (HTML/antora and PDF).

```suggestion
It is recommended to monitor the {salt} event bus in the large scale deployments with xref:specialized-guides:salt/salt-monitoring.adoc#saline[Saline] to identify possible bottlenecks.
```